### PR TITLE
introduce new option that allows reusing args with cargo commands

### PR DESCRIPTION
**Pull request by brotzeit**
@taquangtrung works ?

### CODE REVIEW

File: rustic-cargo.el [hunk 1]
```diff
@@ -65,6 +65,19 @@ If nil then the project is simply created."
   :type 'boolean
   :group 'rustic-cargo)
 
+(defcustom rustic-cargo-use-last-stored-arguments nil
+  "Always rerun cargo commands with stored arguments.
+
+Example:
+When turned off, running `rustic-cargo-test' with `C-u' will store the
+arguments in `rustic-test-arguments' and allows you to reuse them with
+`rustic-cargo-test-rerun'.
```
Comment (taquangtrung) [inline rustic-cargo.el:74]: This description is a bit confusing.

When `rustic-cargo-use-last-stored-arguments` is `nil`, then `rustic-cargo-test` will always use `rustic-default-test-arguments`.

Should the part about `C-u M-x rustic-cargo-test` be described for the function `rustic-cargo-test` but not for this `rustic-cargo-use-last-stored-arguments` variable?

Comment (brotzeit) [reply rustic-cargo.el:74]: > Should the part about C-u M-x rustic-cargo-test be described for the function rustic-cargo-test but not for this rustic-cargo-use-last-stored-arguments variable?

We should apply this variable also for other cargo commands that have rerun commands. I think it's too much if we add a sentence to every command.

Comment (brotzeit) [reply rustic-cargo.el]: Thanks for the review, I applied your suggestion.

Comment (taquangtrung) [reply rustic-cargo.el]: @brotzeit: thanks for your effort! :+1:

**Pull request merged**